rpc/legacyrpc/methods.go

```diff
@@ -861,7 +871,7 @@ func getUnconfirmedBalance(icmd interface{}, w *wallet.Wallet) (interface{}, err
 		return nil, err
 	}
 
-	return (bals.Total - bals.Spendable).ToCoin(), nil
+	return bals.Total - bals.Spendable, nil
```

Comment:
we probably want to redo this part as well (or just remove this RPC completely, it's silly), but the ToCoin here was correct.  Without that this will output atoms instead of the amount in DCR.